packages/react-components/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -27,46 +32,39 @@ const useRootStyles = makeStyles({
     width: '6px',
     height: '6px',
     fontSize: '4px',
+    lineHeight: '4px',
   },
   'extra-small': {
     width: '10px',
     height: '10px',
     fontSize: '6px',
+    lineHeight: '6px',
   },
   small: {
     minWidth: '16px',
     height: '16px',
-    ...shorthands.padding('2px'),
-    ...shorthands.gap('4px'),
```

Comment:
Our current browser support matrix actually supports flex gap

Reply:
I think we're still determining if we need to accommodate browsers that don't support flex gap.

In any case, this still needed to be fixed to accommodate the extra padding that needed to be added around the text but not the icon, and the resulting styles wouldn't be made much simpler by using flex gap. 

Also, the gap here would have been applied to any child elements as well (if the badge contained elements other than just text), which is probably unexpected.